manifests/install.pp

```diff
@@ -14,9 +14,9 @@
 #   are in `file` resources.) Note also that backslashes in double-quoted strings _must_
 #   be escaped and backslashes in single-quoted strings _can_ be escaped.
 class puppet_agent::install (
-  Optional[String]                 $package_version = undef,
+  String                           $package_version = 'present',
   Optional[[Stdlib::Absolutepath]] $install_dir     = undef,
```

Comment:
```diff
-  Optional[[Stdlib::Absolutepath]] $install_dir     = undef,
+  Optional[Stdlib::Absolutepath]   $install_dir     = undef,
```
There's a subtle bug here.. apparently puppet accepts the double brackets but the resulting type will be nil.